core/domain/suggestion_services.py

```diff
@@ -2992,28 +2992,30 @@ def generate_contributor_certificate(
     user_id = user_services.get_user_id_from_username(username)
     if user_id is None:
         raise Exception('There is no user for the given username.')
-    data: Dict[str, Union[str, None]] = {}
+    data: suggestion_registry.ContributorCertificateInfo = (
```

Comment:
Don't do this -- can you drop this line? Shouldn't set it to something invalid.

Reply:
Done
![certificate (32)](https://user-images.githubusercontent.com/41409951/209917361-f0d16ddd-3e0c-4155-993c-8b8e704b3090.jpeg)
![certificate (33)](https://user-images.githubusercontent.com/41409951/209917371-756454eb-36ef-447e-938e-e06de6defeee.jpeg)